### DISCUSSION TITLE
Toggle: have labels based on the checked box

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19630] [Feature Requests]
### Summary

With the Toggle field we currently only have the label: the improvement is to have it accordingly with it's value

### Basic Example

Instead of "Enabled" displayed, in any case,
- Check to enable
- Enabled
- Disabled

### Motivation

Display a message consistent with the value

### Detailed Design

Offer 3 label fields instead of just one

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

Build an extension

### Adoption Strategy

Easy

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
